specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,38 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when a *capacity reservation* is unused.
+
+The CapacityReservationStatus column MUST be present in the billing data when the provider supports *capacity-reservations*. This column MUST be of type String, MUST be null when [CapacityReservationId](#capacityreservationid) is null, and MUST NOT be null when CapacityReservationId is not null and [Charge Category](#chargecategory) is "Usage". The CapacityReservationStatus MUST be one of the allowed values. The CapacityReservationStatus MUST label all unused *capacity reservation* charges and MUST label used *capacity reservation* charges if the provider supports it.
```

Comment:
Attaching a subset of Pricing Category requirements as an example. 

> * PricingCategory MUST be one of the allowed values.
> * PricingCategory MUST be "Standard" when pricing is predetermined at the agreed upon rate for the [billing account](#glossary:billing-account).
> * PricingCategory MUST be "Committed" when [CommitmentDiscountId](#commitmentdiscountid) is not null.

I prefer this wording because it’s more precise and, after all, for consistency with the rest of the spec.

Based on that, here is a new suggestion

```suggestion
The CapacityReservationStatus column adheres to the following requirements:

 * CapacityReservationStatus MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
 * CapacityReservationStatus MUST be null when CapacityReservationId is null and MUST NOT be null when CapacityReservationId is not null.
 * CapacityReservationStatus MUST be one of the allowed values.
* CapacityReservationStatus MUST be "Used" when the charge represents consumption of the *capacity reservation*.
* CapacityReservationStatus MUST be "Unused" when the charge represents the unused portion of the *capacity reservation*.
```